<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatescientist</class>
 <widget class="QDialog" name="updatescientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Scientist</string>
  </property>
  <widget class="QDateEdit" name="updateDeathYear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="maximumDate">
    <date>
     <year>2016</year>
     <month>12</month>
     <day>13</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="updateBirthYear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="maximumDate">
    <date>
     <year>2016</year>
     <month>12</month>
     <day>13</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="updateGenderBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="deathYearLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Died:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateScientistButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthYearLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Born:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateNameInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelUpdateScientistButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="aliveCheckBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="aliveLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="UpdatePictureLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdatePictureButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose an image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fun fact:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateFunFactInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>330</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>updateNameInput</tabstop>
  <tabstop>updateGenderBox</tabstop>
  <tabstop>updateBirthYear</tabstop>
  <tabstop>updateDeathYear</tabstop>
  <tabstop>aliveCheckBox</tabstop>
  <tabstop>updateScientistButton</tabstop>
  <tabstop>cancelUpdateScientistButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
